operations/messages.md

```diff
@@ -30,6 +34,7 @@ Get all messages belonging to the specified [Message threads](messagethreads.md#
 | `Client` | string | required | Name and version of the client application. |
 | `MessageThreadIds` | array of string | required, max 1000 items | Unique identifiers of [Message threads](messagethreads.md#message-thread) from where to return messages. |
 | `CreatedUtc` | [Time interval](messagethreads.md#time-interval) | optional, max length 1 months | Interval in which the [Message](#message) was created. |
+| `Limitation` | [Limitation](messagethreads.md#limitation) | required | Limitation applied to the fetched messages. |

```

Comment:
I would describe this as "Limitation on the quantity of message data returned (using cursor pagination)"